packages/markdown/components/Code/index.jsx

```diff
@@ -0,0 +1,60 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+
+// Only load CodeMirror in the browser, for SSR
+// apps. Necessary because of people like this:
+// https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
+const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
+const copy = require('copy-to-clipboard');
+
+const remapLang = {
+  c: 'cplusplus',
+  'c++': 'cplusplus',
+  cpp: 'cplusplus',
+  docker: 'dockerfile',
```

Comment:
Can you also add this to [syntax-highlighter/modes.js](https://github.com/readmeio/api-explorer/blob/master/packages/syntax-highlighter/modes.js) and add a [test for it](https://github.com/readmeio/api-explorer/blob/master/packages/syntax-highlighter/__tests__/index.test.js#L137-L140) to assert that `docker` returns the same result as `dockerfile`?

And since I added support for a bunch of other aliases, I'm not sure if you just want to make a new `remap` file in the syntax-highlighter package, or pull those in here, or what.